cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -141,17 +141,17 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+start_addon /etc/kubernetes/addons/namespace.yaml 100 10 "" &
+
 # Wait for the default service account
 token_found=""
 while [ -z "${token_found}" ]; do
   sleep .5
-  token_found=$(${KUBECTL} get serviceaccount default -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || true)
+  token_found=$(${KUBECTL} get --namespace=kube-system serviceaccount default -o template -t "{{with index .secrets 0}}{{.name}}{{end}}" || true)
```

Comment:
Sorry - re-reviewing a little more carefully this time.  We really do need a timeout on this retry loop, otherwise it just cycles forever, right?